README.md

```diff
@@ -6,6 +6,7 @@ The Bookstore is a website that allows the user to :
 -Add a book.
```

Comment:
- [ ] [Optional] Netlify checks are not passing, kindly fix this so your page is deployed correctly